src/main/resources/META-INF/plugin.xml

```diff
@@ -255,9 +255,9 @@ L
                 implementationClass="com.redhat.devtools.lsp4ij.features.diagnostics.LSPDiagnosticAnnotator"/>
 
         <!-- LSP textDocument/completion request support -->
-        <typedHandler id="LSPTypedHandlerDelegate"
+        <typedHandler id="LSPCompletionTriggerTypedHandler"
```

Comment:
The other part of this class' rename. Technically the `id` attribute doesn't need to be renamed so, if there is a concern about external references to this ID, I can revert that part and leave the class rename. Let me know if that's the case.